diff --git a/setup.py b/setup.py
index 353f42d..785d624 100755
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,6 @@ setup(
         "Topic :: Scientific/Engineering :: Physics"
     ),
     packages=find_packages(),
-    setup_requires=['pytest-runner'],
     tests_require=['pytest'],
     namespace_packages=['ligo'],
     scripts=[
@@ -62,7 +61,6 @@ setup(
         'astropy-healpix',
         'scipy>=0.7.2',
         'ligo.skymap>=0.1.1',
-        'python-ligo-lw!=1.8.0',
         'lscsoft-glue',
         'lxml',
         'voeventlib>=1.2'
